services/app-proto/src/health_plan_form_data.proto

```diff
@@ -178,6 +178,7 @@ message RateInfo {
   repeated Document rate_documents = 8;
   optional RateCapitationType rate_capitation_type = 9;
   repeated string rate_program_ids = 10;
+  optional string rate_program_name = 11;
   
```

Comment:
`rate_program_name` could be confused for a single program name or a list of program names. Something like `rate_certification_name` or just `rate_name` might describe what this string is. Sorry for not catching this on the previous review.

Reply:
makes sense.  changed to rate_certification_name